Tests/Rules/AvoidUsingAlias.tests.ps1

```diff
@@ -95,5 +100,10 @@ Configuration MyDscConfiguration {
             $violations = Invoke-ScriptAnalyzer -ScriptDefinition "CD" -Settings $settings -IncludeRule $violationName
             $violations.Count | Should -Be 0
         }
+
+        It "do not warn when about Get-* completed cmdlets when the command exists natively on Unix platforms" -skip:(!$IsLinux -and !$IsmacOS) {
```

Comment:
this is a bit confusing - is this better? 
`-skip:( ! ( $IsLinux -or $IsMacOS))`

Reply:
To me both versions are of similar complexity, therefore I don't have a strong opinion on what it should be.